rocketpy/Motor.py

```diff
@@ -137,6 +138,12 @@ def __init__(
             Function. See help(Function). Thrust units are Newtons.
         burnOut : int, float
             Motor burn out time in seconds.
+        distanceNozzlePropellant : int, float
+            Distance from nozzle outlet to the motor reference point, which
+            for Solids and Hybrids is the center of mass of the solid propellant,
+            in meters. Generally positive, meaning a positive position in the
+            z axis which has an origin in the rocket's center of mass (without
+            propellant) and points towards the nose cone.
```

Comment:
Could you elaborate on the choice of using the center of mass of the solid propellant as the reference point, instead of the nozzle itself?

The reason I ask this is the following: all motors have nozzles, but not all motors have grains. Thus, using the nozzle exit as the reference point seems like a more universal choice.